tests/Fixtures/Integration/priority/no_binary_string,set_type_to_cast.test

```diff
@@ -0,0 +1,11 @@
+--TEST--
+Integration of fixers: no_binary_string,set_type_to_cast.
+--RULESET--
+{"no_binary_string": true, "set_type_to_cast": true}
+--EXPECT--
+<?php
+$foo = null;
+
+--INPUT--
+<?php
+settype($foo, b"null");
```

Comment:
omg, that works? like `null` type sure, but binary-string of null as a type? i wouldn't imagine it, ever :(

Reply:
yeah, I had hopes `b"` would be deprecated in 8.2 and removed in 9 to stop this madness : )

Reply:
Maybe time for a PHP RFC? :stuck_out_tongue: